src/components/HomePage/HomePage.js

```diff
@@ -53,6 +55,7 @@ export default function HomePage() {
   ];
 
   const isAdminOrLibrarianRole = (new SessionManager()).getLoggedUser().role != RolesEnum.user;
+  const isLibrarianRole = (new SessionManager()).getLoggedUser().role === RolesEnum.librarian;
```

Comment:
inicjowanie nowego sessionmanagera i pobieranie z niego zalogowanego usera to dosyć wymagająca funkcja (dla pamięci i wydajności), nie powtarzajmy jej jak nie musimy. zamiast tego zrób stałą:
```
const loggedUserRole = new SessionManager()).getLoggedUser().role;
```
i potem jej użyj:
```
const isAdminOrLibrarianRole = loggedUserRole != RolesEnum.user;
const isLibrarianRole = loggedUserRole === RolesEnum.librarian;
```